docs/newbs_git_best_practices.md

```diff
@@ -0,0 +1,16 @@
+# Git Best Practices for Working with QMK
+
+## Or, "How I Learned to Stop Worrying and Love Git."
+
+This section aims to instruct novices in the best ways to have a smooth experience in contributing to QMK. We will walk through the process of contributing to QMK, detailing some ways to make this task easier, and then later we'll break some things in order to teach you how to fix them.
+
+This section assumes a few things:
+
+1. You have a GitHub account, and have [forked the qmk_firmware repository](getting_started_github.md) to your account.
+2. You've [set up your build environment](newbs_getting_started.md?id=environment-setup).
```

Comment:
```suggestion
2. You've [set up your build environment](newbs_getting_started.md?id=set-up-your-environment).
```
Because:
`git log -p 3dec80b77 docs/newbs_getting_started.md`
```
-## Environment Setup
+## Set Up Your Environment
```

Reply:
I think the following is better:

```suggestion
2. You've [set up your build environment](newbs_getting_started.md?id=set-up-qmk).
```

Reply:
Well spotted @mtei. I think we probably need to specify the user needs the build environment and QMK both.